packages/@aws-cdk/aws-backup/README.md

```diff
@@ -85,3 +85,26 @@ const vault = new BackupVault(stack, 'Vault', {
 
 A vault has a default `RemovalPolicy` set to `RETAIN`. Note that removing a vault
 that contains recovery points will fail.
+
+
+## Importing existing backup vault
+
+To import an existing backup vault into your CDK application, use the `BackupVault.fromBackupVaultArn` or `BackupVault.fromBackupVaultName` 
+factory method. Here is an example of giving a lambda role permission to start a backup job:
+
+```ts
+import * as lambda from '@aws-cdk/aws-lambda';
+import { Stack } from '@aws-cdk/core';
+
+let stack = new Stack();
```

Comment:
I appreciate the complete example here, but you can just use `this` as the scope. If you'd like to ensure compilation, please create a [default fixture](https://github.com/aws/aws-cdk/blob/master/CONTRIBUTING.md#rosetta) that sets up a stack